.github/ISSUE_TEMPLATE/experimental_feature.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Experimental Feature Requirements
+about: Requirements for experimental features
+---
+
+This page lists the requirements for experimental features. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+- [ ] Feature is reviewed in a TOC meeting
+- [ ] Design document has been written
+- [ ] Feature is disabled by default
+- [ ] Steps for enabling the feature have been documented
+- [ ] Plan exists for getting feedback
```

Comment:
Would be good to provide guideline to get feedback.  Is it via our preview profile?

Reply:
I think it varies depending on feature right?

* For, say `istioctl experimental...`, this might be just to run a command from istioctl and provide feedback
* for other features, we've discussed in TOC having users compile their own versions of Istio to test a feature
* for other features, it's a preview profile element or enabling a specific field in their installation CRD

I'm not sure we can be too specific on how all features have users test something/get feedback, but asking for a description of how they want to approach that here makes sense to me.